packages/shared/src/models/IContainerConnectionInfo.ts

```diff
@@ -27,11 +27,12 @@ export interface ContainerProviderConnectionInfo {
 }
 
 export interface CheckContainerConnectionResourcesOptions {
-  modelInfo: ModelCheckerInfo;
+  model: ModelInfo & { memory: number };
+  context: ModelCheckerContext;
   connection?: ContainerProviderConnectionInfo;
 }
 
-export type ContainerConnectionInfo =
+export type ContainerConnectionResourceInfo =
```

Comment:
i don't get why it was renamed as only one of the subtypes directly refers to resources.  So in most cases there are no info about resources

Reply:
I revert the rename